gen/calc.py

```diff
@@ -387,7 +387,7 @@ def calculate_config_yaml(user_arguments):
 def calculate_mesos_isolation(enable_gpu_isolation):
     isolators = ('cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,'
                  'docker/runtime,docker/volume,volume/sandbox_path,posix/rlimits,'
-                 'namespaces/pid,com_mesosphere_MetricsIsolatorModule')
+                 'namespaces/pid,linux/capabilities,com_mesosphere_MetricsIsolatorModule')
```

Comment:
As I mentioned in the summary, this change enables the isolator, but does not set the agent flags. Therefore, it'll be like a no-op.

I can add a test using 'mesos-execute' though.